### PR TITLE
Reader: Add reader coldstart demo page

### PR DESCRIPTION
**Pull request by gibrown**
We need a page for evaluation of the cold start and warm start algorithms side by side. This should make it easier to find problems and discuss what needs to be improved.

Test live: https://calypso.live/?branch=add-reader-coldstart-demo

### CODE REVIEW

**Comment by bluefuton** [comment]
Can we give it a squash 🍊 to roll the eslint fixes into the main commit?

**Comment by blowery** [comment]
@bluefuton tangent, but you can squash when merging now. Hit the arrow on the right side of the Confirm button and pick "Squash and Merge"

File: client/reader/sidebar/index.jsx [hunk 1]
```diff
@@ -186,6 +186,19 @@ const ReaderSidebar = React.createClass( {
 						}
 
 						{
+							// Post Recommendations Cold Start - Used by the Data team to test cold start algorithms
+							config.isEnabled( 'reader/recommendations/posts' ) &&
+							(
+								<li className={ ReaderSidebarHelper.itemLinkClass( '/recommendations/cold', this.props.path, { 'sidebar-streams__post-recommendations': true } ) }>
+									<a href="/recommendations/cold">
+										<Gridicon icon="star" size={ 24 } />
+										<span className="menu-link-text">{ this.translate( 'Coldstart (Alpha)' ) }</span>
```
Comment (bluefuton) [inline client/reader/sidebar/index.jsx]: Should it have the alpha label if we've released it? :)

File: client/reader/recommendations/controller.js [hunk 2]
```diff
@@ -43,10 +43,17 @@ export default {
...
 	recommendedPosts( context ) {
 		var RecommendedPostsStream = require( 'reader/recommendations/posts' ),
-			basePath = route.sectionify( context.path ),
-			fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Recommended Posts',
-			RecommendedPostsStore = feedStreamFactory( 'recommendations_posts' ),
-			mcKey = 'recommendations_posts';
+			basePath = route.sectionify( context.path );
+
+		if ( '/recommendations/posts' === basePath ) {
+			var	fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Recommended Posts',
```
Comment (bluefuton) [inline client/reader/recommendations/controller.js]: Let's make these `const`.

File: client/reader/sidebar/index.jsx (hunk 1 above)
Comment (blowery) [inline client/reader/sidebar/index.jsx]: only released to wpcalypso

File: client/reader/recommendations/controller.js (hunk 2 above)
Comment (TooTallNate) [inline client/reader/recommendations/controller.js]: style nit: strange use of tab after `var`

File: client/reader/sidebar/index.jsx (hunk 1 above)
Comment (bluefuton) [inline client/reader/sidebar/index.jsx]: Oh I see - I thought it was presenting regular Cold Start. Which it isn't :)

**Comment by gibrown** [comment]
@bluefuton I think that addresses everything.

**Comment by bluefuton** [comment]
Looks good to me 🚢

**Pull request merged**